ProcessMaker/processmaker
# Epic/FOUR-11531: [PRD] Process LaunchPad IT2

## Issue & Reproduction Steps
Process LaunchPad
https://processmaker.atlassian.net/wiki/spaces/PM4/pages/3232301057/Process+LaunchPad

https://www.figma.com/file/bI3YLt9n2771ReExAQIggs/Winter24?type=design&mode=design&t=3rPEwvG6sK5NdKXv-0

## Solution
**11. Designer menu**
As a Designer, I want to see a menu with designer options (ellipsis from process list) including editing the launchpad.
	
**12. Launchpad setup**
As process author, after first publishing the process, I want the option to configure the launchpad (icon, thumbnail, grouped by chart).
This option Launchpad should go in the initial configuratioon of the process.

## How to Test
1. Enter a process to access its Launchpad.
3. See a menu with Designer options (ellipsis from process list)

## Related Tickets & Packages
- [Link to any related FOUR tickets, PRDs, or packages](https://processmaker.atlassian.net/browse/FOUR-11531)

## Code Review Checklist

ci:deploy
ci:next
ci:package-versions:feature/FOUR-11531
ci:package-analytics-reporting:bugfix/FOUR-12650
.
